apps/recipes-react-components/package.json

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27100&commitId=d573a813e3df91e9d72d697c193233552db76de7&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1864 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|19| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|42| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|21| Changed|
|Avatar Converged.basic.normal.chromium.png|21| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
These changes are intentional fixes, resolving.